website/content/commands/connect/envoy.mdx

```diff
@@ -136,12 +136,24 @@ proxy configuration needed.
   `-register`. This takes the form of `<ip address>:<port>` but also supports go-sockaddr
   templates.
 
+  For terminating and mesh gateways, this is the IP and port on which
+  to receive traffic from services in the mesh. For ingress gateways, this is
+  the IP and port at which to instantiate a `/ready` HTTP endpoint which is used
+  by Consul for health checking the gateway. The specified IP will also be used
```

Comment:
```suggestion
  If Envoy is configured as a terminating or mesh gateway, traffic from services within the mesh will be received at the specified IP and port. 
  
  If Envoy is configured as an ingress gateway, a `/ready` HTTP endpoint will be instantiated at the specified IP and port. Consul uses `/ready` HTTP endpoints to check gateway health. The specified IP will also be used
```
We generally avoid passive voice, but in this instance I think using PV improves readability. Your original text was grammatically sound, so feel free to ignore this suggestion if you disagree.